### PR TITLE
Always push a .qcow2 image to artifacts

### PR DESCRIPTION
**Pull request by johnbieren**
Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

**Comment by p3ck** [comment]
@arilivigni - this could have a huge impact on the storage we use on artifacts.ci.centos.org.  Has anyone calculated how much space this could end up using?  And has anyone confirmed this will be ok with CentOS?

File: Jenkinsfile [hunk 1]
```diff
@@ -482,7 +471,7 @@ podTemplate(name: podName,
                     pipelineUtils.getContainerLogsFromPod(OPENSHIFT_NAMESPACE, env.NODE_NAME)
 
                     // Archive our artifacts
-                    step([$class: 'ArtifactArchiver', allowEmptyArchive: true, artifacts: '**/logs/**,*.txt,*.groovy,**/job.*,**/*.groovy,**/inventory.*', excludes: '**/job.props,**/job.props.groovy,**/*.example', fingerprint: true])
+                    step([$class: 'ArtifactArchiver', allowEmptyArchive: true, artifacts: '**/logs/**,*.txt,*.groovy,**/job.*,**/*.groovy,**/inventory.*', excludes: '**/job.props,**/job.props.groovy,**/*.example,**/*.qcow2', fingerprint: true])
```
Comment (scoheb) [inline Jenkinsfile]: Are we sure we want to keep qcow files as Jenkins artifacts?

Comment (johnbieren) [reply Jenkinsfile]: I thought adding it to excludes: would make it so it didn't save as an artifact, not the opposite

Comment (scoheb) [reply Jenkinsfile]: Oops. My bad. You are right. Carry on!

**Comment by arilivigni** [comment]
@p3ck and @johnbieren as we spoke this morning let's wait on this until we have block storage

**Pull request closed without merging**